org.lflang/src/org/lflang/util/FileUtil.java

```diff
@@ -378,19 +378,6 @@ private static void copyDirectoryFromJar(JarURLConnection connection, final Path
         }
     }
 
-    /**
-     * Copy a list of files from a given source directory to a given destination directory.
-     * @param srcDir The directory to copy files from.
-     * @param dstDir The directory to copy files to.
-     * @param files The list of files to copy.
-     * @throws IOException If any of the given files cannot be copied.
-     */
-    public static void copyFilesFromClassPath(String srcDir, Path dstDir, List<String> files) throws IOException {
```

Comment:
I guess we no longer copy lists of individual files from the class path?

Reply:
Yes, that was an objective of this PR